wmi_check/datadog_checks/wmi_check/wmi_check.py

```diff
@@ -36,7 +36,7 @@ def check(self, instance):
         filters = instance.get('filters')
         tag_by = instance.get('tag_by', "")
         tag_queries = instance.get('tag_queries', [])
-        constant_tags = instance.get('constant_tags')
+        constant_tags = instance.get('tags', [])
```

Comment:
If you want to do this, you can't just remove `constant_tags` in case someone's using it.  Get the tags, but also check for `constant_tags`, post a warning message that `constant_tags` is being deprecated and they should use `tags`

Reply:
Ohh, sounds good! I'll fix it :)